TCAT/Controllers/FavoritesViewController.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  FavoritesViewController.swift
+//  TCAT
+//
+//  Created by Lucy Xu on 11/4/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import Pulley
+import UIKit
+
+class FavoritesViewController: UIViewController {
+
+    // MARK: - View vars
+    private var editButton: UIButton!
+    private let favoritesBlueColor = UIColor(hex: "08A0E0")
```

Comment:
we only use this in `setupViews()` so we can just put it in the function instead of making this a class variable